resolution-aware-images.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Resolution Aware Images
+permalink: /resolution-aware-images/
+---
+
+Flutter can load resolution-appropriate images for the current device
+pixel ratio.
+
+*Tip*: Remember to declare all of your resources inside `flutter.yaml`,
+as shown in the [Tutorial](/tutorial/).
```

Comment:
When flutter builds an app it creates a bundle of assets that are accessible through the class AssetBundle (and the widget DefaultAssetBundle).  Maybe we should have separate documentation on that somewhere?